runtime/msgpack-serialization.h

```diff
@@ -53,17 +53,25 @@ inline string f$msgpack_serialize_safe(const T &value) noexcept {
   }
   return res.val();
 }
-
 template<class InstanceClass>
-inline Optional<string> f$instance_serialize(const class_instance<InstanceClass> &instance) noexcept {
+inline Optional<string> f$common_instance_serialize(const class_instance<InstanceClass> &instance, string *out_err_msg) noexcept {
```

Comment:
`f$` at start of the name means there's such a function in `functions.txt`, but it's not the case.
Let's rename it to smth like `instance_serialize_impl`